.env

```
@@ -0,0 +1,12 @@
+MYSQL_ROOT_PASSWORD=test@123
+MYSQL_DATABASE=employee_management
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The MYSQL_DATABASE key should go before the MYSQL_ROOT_PASSWORD key

(UnorderedKey)